Common/Python/PandasConverter.cs

```diff
@@ -346,7 +387,18 @@ public PyObject ToPandasDataFrame(dynamic pandas, int levels = 2)
                 var index = pandas.MultiIndex.from_tuples(tuples, names: names.Split(','));
 
                 // Returns a dictionary keyed by column name where values are pandas.Series objects
-                return _series.ToDictionary(k => k.Key, v => pandas.Series(v.Value, index));
+                var pandasSeries = new Dictionary<string, dynamic>();
+                foreach (var kvp in _doubleSeries)
+                {
+                    if (kvp.Value.Count != tuples.Length) continue;
```

Comment:
If this does happen what does that mean? In what case would this condition hold true and are we hiding a bug by continuing in this case?

Reply:
When we create a pandas Series in the following line, it is required that the length of those list are the same.
This check was created because ticks that are of type trade does not have bid and ask prices, so I am not adding zeroes to the dictionary and, consequently, I am not adding that information to the data frame